src/docs/product/cli/releases.mdx

```diff
@@ -18,8 +18,8 @@ Because releases work on projects you will need to specify the organization and
 
 Releases are created with the `sentry-cli releases new` command. It takes at the very least a version identifier that uniquely identifies the releases. There are a few restrictions -- the release name cannot:
 
-- contain newlines or spaces
-- use a forward slash ("/"), back slash ("\"), period ("."), or double period ("..")
```

Comment:
It also cannot start or end with a spaces.

Reply:
I think for starting and ending with string is safer to trim the string on the CLI side. This change is mostly to keep the documentation consistent with the SDK release guide.